File: src/Atividade2/Exercicio4.java
package Atividade2;

import java.util.Scanner;

public class Exercicio4 {
    public static void main(String[] args) {
        Scanner ler = new Scanner(System.in);

        int[] vetor = new int[20];
        int i;
        int num;

        System.out.println("Digite 20 numeros inteiros");

        for(i=0; i<20; i++){
            System.out.println("Digite o vetor " + i);
            vetor[i] = ler.nextInt();
        }

        for(i=0; i<10; i++){
            num=vetor[i];
            vetor[i]=vetor[19-i];
            vetor[19-i]=num;
        }

        for(i=0; i<20; i++){
            System.out.println("Novo Vetor = " +vetor[i]);
        }

    }
}

File: src/LPII/Atividade1/Exercicio4.c
package LPII.Atividade1;

public class Exercicio4 {
    #include <stdio.h>
    #define v 20

    int main(){

        int vetor[v], cont, num;

        printf("Digite 20 numeros inteiros\n\n");

        for(cont=0; cont<v; cont++){
            printf("Digite o vetor %d\n", cont);
            scanf("%d", &vetor[cont]);
        }

        for(cont=0; cont<10; cont++){
            num=vetor[cont];
            vetor[cont]=vetor[v-cont];
            vetor[v-cont]=num;
        }

        for(cont=0; cont<v; cont++){
            printf("Novo Vetor de [%d] = %d\n",cont, vetor[cont+1]);
        }

        return 0;
    }
}
